tools/python/transit/transit_color_palette.py

```diff
@@ -0,0 +1,34 @@
+transit_palette = ["000000",
+                   "ff0000",
+                   "00ff00",
+                   "0000ff"]
+
+
+def to_rgb(color_str):
+    if len(color_str) != 6:
+        return (0, 0, 0)
+    r = int('0x' + color_str[0:2], 16)
+    g = int('0x' + color_str[2:4], 16)
+    b = int('0x' + color_str[4:], 16)
+    return (r, g, b)
+
+
+class Palette:
+    def __init__(self, colors):
+        self.colors = {}
+        for color in colors:
+            self.colors[color] = to_rgb(color)
+
+    def get_nearest_color(self, color_str):
+        """Returns the nearest color from the palette."""
+        nearest_color = None
+        color = to_rgb(color_str)
+        min_diff = None
+        for palette_color in self.colors.values():
```

Comment:
Why do you need a dict instead of a sequence?

Reply:
There was no need indeed.